.gitignore

```diff
@@ -1,5 +1,32 @@
-# used for development
-node_modules
+# anything, anywhere starting with __ or ~~
+__*
+~~*
+
+# Local development
+*.env
+*.dev
+
+# Apple
```

Comment:
I recently discovered global gitignore https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer
imho it is a cleaner way to exclude machine specific files

Reply:
I'm actually pulling to the other direction (for now), avoiding anything global (e.g. packages). The goal is to get to a point where node development can be done on a factory reset Mac with minimal configuration (git + docker +  ??? ). There is some more WIP here as there are more repos involved (testing).